edbdeploy/ansible.py

```diff
@@ -15,8 +15,8 @@ class AnsibleCli:
     def __init__(self, dir, bin_path=None):
         self.dir = dir
         # Ansible supported versions interval
-        self.min_version = (0, 0, 0)
-        self.max_version = (2, 10, 7)
+        self.min_version = (2, 10, 8)
+        self.max_version = (9, 9, 9)
```

Comment:
`max_version` must be set to the most recent tested and approved version because the sub-command `setup` install the `max_version`.